src/v3/bandwidthRtc.ts

```diff
@@ -505,6 +525,22 @@ class BandwidthRtc {
       this.publishedStreams.delete(stream.mediaStream.id);
     }
   }
+
+  /**
+   * Can be overridden in environments where RTCPeerConnection is not natively present
+   * @returns new RTCPeerConnection
+   */
+  private createPeerConnection() {
+    return new RTCPeerConnection(RTC_CONFIGURATION);
+  }
+
+  /**
+   * Can be overridden in environments where MediaStream is not natively present
+   * @returns true if input is a MediaStream, false otherwise
+   */
+  private isMediaStream(input: MediaStreamConstraints | MediaStream) {
+    return input instanceof MediaStream;
+  }
```

Comment:
Added these stupid simple helpers to allow dependency injection from the trogdor test; I don't _love_ it, but it's a whole lot better than what I was doing passing a dict of primitives into the constructor and setting fields on `global`.